test/feed_test.exs

```diff
@@ -35,6 +35,19 @@ defmodule FeedTest do
     assert length(list) == 25
   end
 
+  test "parser includes article author" do
+    [
+      {"elixirlang", "José Valim"},
+      {"latimes", "Randall Roberts"},
+      {"technewsworld", "Rob Enderle"},
```

Comment:
This also passed with the following extra entries, which I had in place originally to show that the new behavior didn't break on feeds without authorship information:

```elixir
{"ntv", nil},
{"spiegel", nil},
```